spec/classes/init_spec.rb

```diff
@@ -13,11 +13,29 @@
           facts
         end
 
-        context 'with default parameters' do
```

Comment:
What's wrong with the default parameter context? The contexts are supposed to read like a sentence.